crates/sui-core/src/node_sync/node_follower.rs

```diff
@@ -28,7 +28,7 @@ use tracing::{debug, info, trace, warn};
 
 // How long must a follower task run for for us to clear the exponential backoff for that peer?
 const CLEAR_BACKOFF_DURATION: Duration = Duration::from_secs(30);
-const MIN_RECONNECTION_DELAY: Duration = Duration::from_millis(500);
+const MIN_RECONNECTION_DELAY: Duration = Duration::from_millis(1500);
```

Comment:
Right now fullnodes are limited to receiving 2000 digests per second, if they receive 1000 instantly and then sleep for 500ms. This makes that bottleneck even more severe. We probably need to substantially increase `NUM_ITEMS_PER_REQUEST` if we want nodes to be able to catch up reasonably quickly.